search/conf.py

```diff
@@ -0,0 +1,29 @@
+"""search application configuration"""
+from appconf import AppConf
+from django.conf import settings
+
+
+class SearchAppConf(AppConf):
+    """Configuration class for a configurable CAS authentication system"""
+
+    # ENABLED = False
```

Comment:
I don't understand what "this" means here (the "this" I want to call `HAYSTACK_URL`).